server/jamovi/server/i18n.py

```diff
@@ -0,0 +1,55 @@
+
+import os.path
+import json
+
+from .utils import conf
+
+
+class I18n:
+    def __init__(self):
+        self.i18n_msgs = None
+        self._code = None
+        self._failed = False
+
+    @property
+    def code(self):
+        return self._code
+
+    def set_language(self, code):
+        if self._code != code:
+            self._code = code
+            self.i18n_msgs = None
+            self._failed = False
+
+    def _load_translations(self):
```

Comment:
rather than returning True/False, let's make this return the actual translations, and then you can mark this with `@lru_cache`

https://docs.python.org/dev/library/functools.html#functools.lru_cache